components/brave_wallet_ui/components/desktop/popup-modals/partners_consent_modal/partners_consent_modal.tsx

```diff
@@ -51,18 +53,23 @@ export function PartnersConsentModal(
   )
 
   const onClickTermsOfUse = () => {
-    chrome.tabs.create(
-      {
-        url: 'https://www.meld.io/terms-of-use'
-      },
-      () => {
-        if (chrome.runtime.lastError) {
-          console.error(
-            'tabs.create failed: ' + chrome.runtime.lastError.message
-          )
+    if (chrome.tabs !== undefined) {
+      chrome.tabs.create(
+        {
+          url: MELD_TERMS_OF_USE_URL
+        },
+        () => {
+          if (chrome.runtime.lastError) {
+            console.error(
+              'tabs.create failed: ' + chrome.runtime.lastError.message
+            )
+          }
         }
-      }
-    )
+      )
+      return
+    }
+    // Tabs.create is desktop specific. Using window.open for android.
+    window.open(MELD_TERMS_OF_USE_URL, '_blank', 'noopener noreferrer')
```

Comment:
Can we use `window.open` on desktop too? It feels like that would be simplest.

Reply:
This UI is also used in the `Wallet` panel, `window.open` does not work in that case.